File: BackEnd/Models/Dictionary/Unit.cs
using Common.Enum;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class Unit : BaseModel
    {
        public Guid UnitID { get; set; }
        public string UnitName { get; set; }
        public bool Status { get; set; }
        public string Description { get; set; }
    }
}

File: FrontEnd/MshopUI/src/app/models/unit.ts
import { BaseModel } from './base-model/base-model';
import { Status } from '../services/common/enumeration/enumeration';

export class Unit extends BaseModel{
    UnitID: string;
    UnitName: string = "";
    Status: Status = 0;
    Description: string = "";
    constructor(){
        super();
    }
}